dask/array/core.py

```diff
@@ -3757,12 +3757,23 @@ def concatenate3(arrays):
            [1, 2, 1, 2],
            [1, 2, 1, 2]])
     """
+    from .utils import IS_NEP18_ACTIVE
+
     arrays = concrete(arrays)
     if not arrays:
         return np.empty(0)
 
     advanced = max(core.flatten(arrays, container=(list, tuple)),
                    key=lambda x: getattr(x, '__array_priority__', 0))
+
+    if if (IS_NEP18_ACTIVE and not all(np.ndarray.__array_function__ is
```

Comment:
```suggestion
    if (IS_NEP18_ACTIVE and not all(np.ndarray.__array_function__ is
```

Reply:
Sorry, my laptop’s touchpad is way too sensitive and automatically pastes stuff.